crypto/objects/o_names.c

```diff
@@ -118,7 +132,10 @@ int OBJ_NAME_new_index(unsigned long (*hash_func) (const char *),
         name_funcs->cmp_func = cmp_func;
     if (free_func != NULL)
         name_funcs->free_func = free_func;
-    return (ret);
+
+    out:
+        CRYPTO_THREAD_unlock(lock);
+        return (ret);
```

Comment:
Too much indentation here! Indent the label one space and the other lines 4 spaces
Also the preferred style is now to not put () around a return expression (where we touch code that has that we remove it).